File: Aplikacija/EstateExplorer/EstateExplorer/ClientApp/src/models/Stan.js
import Nekretnina from "./Nekretnina";

class Stan extends Nekretnina {
    povrsina;
    brojSoba;
    cenaKv;
    sprat;
    brojUlaza;
    orijentacija;
    opis;

    constructor() {
        super();
        this.povrsina = 0;
        this.brojSoba = 0;
        this.cenaKv = 0;
        this.sprat = 0;
        this.brojUlaza = 0;
        this.orijentacija = "";
        this.opis = "";
    }
}

export default Stan;

File: Aplikacija/EstateExplorer/EstateExplorer/Models/Stan.cs
using System;
using System.ComponentModel.DataAnnotations;

namespace EstateExplorer.Models
{
	public class Stan : Nekretnina
	{
		
		public double BrojSoba { get; set; }
		public double CenaPoKvadratuBezPDV { get; set; }
		public int Sprat { get; set; }
		public string BrojUlaza { get; set; } = default!;
		public string Orijentacija { get; set; }=default!;
		public string? Opis { get; set; }
		
	}
}
